### PR TITLE
[RFC] vim-patch:7.4.740

### PR DESCRIPTION
**Pull request by jbradaric**
```
Problem:    ":1quit" works like ":.quit". (Bohr Shaw)
Solution:   Don't exit Vim when a range is specified. (Christian Brabandt)
```

https://github.com/vim/vim/commit/c7a0d32c834ff321491d8fd7bc68bf6cabd11e44

### CODE REVIEW

**Comment by jbradaric** [comment]
It appears that the test fails because help tags are not available.

**Comment by justinmk** [comment]
@jbradaric Could you try `new|set filetype=help` instead?

**Comment by justinmk** [comment]
Actually `set filetype=help` is not equivalent--just tried it locally. `set buftype=help` _is_ equivalent. Sorry 'bout that.

File: src/nvim/testdir/test13.in [hunk 1]
```diff
@@ -48,6 +48,12 @@ otestje3
 :au BufWipeout Xtestje1 buf Xtestje1
 :bwipe
 :w >>test.out
+:only
+:new|set ft=help
```
Comment (justinmk) [inline src/nvim/testdir/test13.in]: :new|set buftype=help

**Comment by justinmk** [comment]
Merged after partial squash. Verified that test fails before patch. Thanks!

**Pull request closed without merging**